broker/src/main/java/io/camunda/zeebe/broker/system/configuration/DataCfg.java

```diff
@@ -42,6 +42,9 @@ public final class DataCfg implements ConfigurationEntry {
   @Override
   public void init(final BrokerCfg globalConfig, final String brokerBase) {
     directory = ConfigurationUtil.toAbsolutePath(directory, brokerBase);
+    if (runtimeDirectory != null) {
+      runtimeDirectory = ConfigurationUtil.toAbsolutePath(runtimeDirectory, brokerBase);
+    }
```

Comment:
@npepinpe This was added after your review. I think it makes sense to add this to be consistent with the data directory configuration.

Reply:
Good catch!